src/apollo/server.js

```diff
@@ -0,0 +1,34 @@
+const { ApolloServer } = require('apollo-server-express');
+const fetch = require('node-fetch');
+const { introspectSchema, wrapSchema } = require('graphql-tools');
+const { print } = require('graphql');
+const { graphqlUrl, getValidAccessToken, realmAvailable } = require('../util/RealmClient');
+
+const executor = async ({ document, variables, _context }) => {
```

Comment:
The pattern can be found [here.](https://www.graphql-tools.com/docs/remote-schemas)

Reply:
I might leave a comment in code so I remember where this came form.

Reply:
Added.